src/DetectChanges/BCBreak/FunctionBased/ParameterDefaultValueChanged.php

```diff
@@ -38,8 +41,23 @@ public function __invoke(
         $changes = Changes::empty();
 
         foreach (Dict\intersect_by_key($fromParametersWithDefaults, $toParametersWithDefaults) as $parameterIndex => $parameter) {
-            $defaultValueFrom = $parameter->getDefaultValue();
-            $defaultValueTo   = $toParametersWithDefaults[$parameterIndex]->getDefaultValue();
+            try {
+                $defaultValueFrom = $parameter->getDefaultValue();
+                $defaultValueTo   = $toParametersWithDefaults[$parameterIndex]->getDefaultValue();
+            } catch (UnableToCompileNode $unableToCompileNode) {
+                $parameterDefaultExpression   = $parameter->getDefaultValueExpression();
+                $toParameterDefaultExpression = $toParametersWithDefaults[$parameterIndex]->getDefaultValueExpression();
+
+                if (
+                    $toParameterDefaultExpression instanceof Expr &&
+                    $parameterDefaultExpression instanceof  Expr &&
+                    $toParameterDefaultExpression->getType() === $parameterDefaultExpression->getType()
+                ) {
+                    continue;
```

Comment:
Let's rewrite this into a `private` function perhaps? Gets rid of the psalm issue too, I suppose.

Reply:
Tried it right now, but this will not remove the the psalm issues. But i will still keep it in the private method.